src/Integrations/Moments/SettingsMoments.php

```diff
@@ -148,59 +150,84 @@ public function getSettingsGlobalData(): array
+									'introSubtitle' => \__('To ensure your form is not lost, it is important to activate the "Stop form syncing" option in the debug settings and avoid clicking on the form sync button.', 'eightshift-forms'),
+									'introIsHighlighted' => true,
+									'introIsHighlightedImportant' => true,
 								],
-								'additionalClass' => Components::getComponent('form')['componentTestApiJsClass'] . ' es-submit--api-test',
-							],
+							] : [
+								[
+									'component' => 'input',
+									'inputName' => $this->getSettingsName(self::SETTINGS_MOMENTS_API_URL_KEY),
+									'inputFieldLabel' => \__('API url', 'eightshift-forms'),
+									'inputFieldHelp' => $this->getGlobalVariableOutput('ES_API_URL_MOMENTS', !empty($apiUrl)),
+									'inputType' => 'text',
+									'inputIsRequired' => true,
+									'inputValue' => !empty($apiUrl) ? $apiUrl : $this->getOptionValue(self::SETTINGS_MOMENTS_API_URL_KEY),
+									'inputIsDisabled' => !empty($apiUrl),
+								],
+								[
+									'component' => 'input',
+									'inputName' => $this->getSettingsName(self::SETTINGS_MOMENTS_API_KEY_KEY),
+									'inputFieldLabel' => \__('API key', 'eightshift-forms'),
+									'inputFieldHelp' => $this->getGlobalVariableOutput('ES_API_KEY_MOMENTS', !empty($apiKey)),
+									'inputType' => 'password',
+									'inputIsRequired' => true,
+									'inputValue' => !empty($apiKey) ? 'xxxxxxxxxxxxxxxx' : $this->getOptionValue(self::SETTINGS_MOMENTS_API_KEY_KEY),
+									'inputIsDisabled' => !empty($apiKey),
+								],
+								[
+									'component' => 'input',
+									'inputName' => $this->getSettingsName(self::SETTINGS_MOMENTS_API_USERNAME_KEY),
+									'inputFieldLabel' => \__('API username', 'eightshift-forms'),
+									'inputFieldHelp' => $this->getGlobalVariableOutput('ES_API_USERNAME_MOMENTS', !empty($apiUsername)),
+									'inputType' => 'text',
+									'inputIsRequired' => true,
+									'inputValue' => !empty($apiUsername) ? $apiUsername : $this->getOptionValue(self::SETTINGS_MOMENTS_API_USERNAME_KEY),
+									'inputIsDisabled' => !empty($apiUsername),
+								],
+								[
+									'component' => 'input',
+									'inputName' => $this->getSettingsName(self::SETTINGS_MOMENTS_API_PASSWORD_KEY),
+									'inputFieldLabel' => \__('API password', 'eightshift-forms'),
+									'inputFieldHelp' => $this->getGlobalVariableOutput('ES_API_PASSWORD_MOMENTS', !empty($apiPassword)),
+									'inputType' => 'password',
+									'inputIsRequired' => true,
+									'inputValue' => !empty($apiPassword) ? 'xxxxxxxxxxxxxxxx' : $this->getOptionValue(self::SETTINGS_MOMENTS_API_PASSWORD_KEY),
+									'inputIsDisabled' => !empty($apiPassword),
+								],
+								[
+									'component' => 'divider',
+									'dividerExtraVSpacing' => true,
+								],
+								[
+									'component' => 'submit',
+									'submitFieldSkip' => true,
+									'submitValue' => \__('Test api connection', 'eightshift-forms'),
```

Comment:
```suggestion
									'submitValue' => \__('Test API connection', 'eightshift-forms'),
```

Reply:
this button is on multiple locations so I will rename it on all in the next PR tnx for the comment